docs/source/en/optimization/fp16.md

```diff
@@ -296,3 +307,7 @@ An input is projected into three subspaces, represented by the projection matric
 ```py
 pipeline.fuse_qkv_projections()
 ```
+
+## Resources
```

Comment:
Will support https://github.com/huggingface/flux-fast/pull/10 and Kontext is already supported.